rubicon_ml/cli.py

```diff
@@ -78,7 +80,78 @@ def ui(root_dir, page_size, project_name, host, port, debug, storage_options):
     dashboard.serve(run_server_kwargs=run_server_kwargs)
 
 
-# CLI groups
+@cli.command(
+    context_settings=dict(
+        ignore_unknown_options=True,
+    )
+)
+@click.option(
+    "--root-dir",
+    type=click.STRING,
+    default=os.environ.get("RUBICON_ROOT_DIR"),
+    help="The absolute path to the top level folder holding the rubicon-ml project.",
+    required=False,
+)
+@click.option(
+    "--project-name",
+    default=os.environ.get("RUBICON_PROJECT_NAME"),
+    type=click.STRING,
+    help="Name of rubicon project to retrieve experiments from",
+    required=False,
+)
+@click.option(
+    "--color",
+    "-c",
+    type=click.STRING,
+    help="Color of query output",
+    required=False,
+)
+@click.argument(
+    "query",
+    nargs=1,
+    required=True,
+)
+def search(root_dir, project_name, color, query):
+    """
+    This function provides capability to query  rubicon experiments from the command line without using python
+    directly and output the JSON in the command line.
+    """
+
+    proj_name_env = "RUBICON_PROJECT_NAME"
+    root_dir_env = "RUBICON_ROOT_DIR"
+    fg_color = "bright_yellow"
+
+    if (proj_name_env not in os.environ and project_name is None) or (
+        root_dir_env not in os.environ and root_dir is None
+    ):
+        warnings.warn(
```

Comment:
since we're returning after raising this, I think I'd argue it's more than a warning. based on a quick search it seems like using `click.secho` for the message like you do later on and then `sys.exit` instead of `return` would be the preferred thing here so we can exit with an error code
> https://github.com/pallets/click/issues/747